modules/drivers/video/driver.cc

```diff
@@ -0,0 +1,75 @@
+/******************************************************************************                                                                                                                              
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#include "modules/drivers/video/input.h"
+#include "modules/drivers/video/driver.h"
+
+namespace apollo {
+namespace drivers {
+namespace video {
+
+
+using apollo::drivers::video::config::CameraH265Config ;
+using apollo::drivers::CompressedImage;
+
+CameraDriver::CameraDriver(CameraH265Config& h265_cfg) 
+{
```

Comment:
Please run "./apollo.sh lint" to make sure it passes the CI lint check.

Reply:
agree, done